Code/GraphMol/.vscode/settings.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Please remove this file. I guess it was added accidentally

Reply:
done